bleachbit/GUI.py

```diff
@@ -428,8 +426,9 @@ def about(self, __event):
         dialog.set_comments(_("Program to clean unnecessary files"))
         dialog.set_copyright("Copyright (C) 2008-2017 Andrew Ziem")
         try:
-            dialog.set_license(open(license_filename).read())
-        except:
+            with open(bleachbit.license_filename) as f:
```

Comment:
This is in the wrong PR, but still a good idea. Otherwise we leak a file descriptor every time a user opens the about dialog